docs/getting-started/ers-req.asciidoc

```diff
@@ -35,8 +38,25 @@ Follow these guidelines to ensure clusters have adequate memory to handle data v
 * With 1GB of JVM heap, the risk scoring engine can safely process around 20 million documents, or 30 days of risk data with an ingest rate of around 450 documents per minute.
 
 [discrete]
-== Known limitations
+=== Known limitations
 
 * You can only enable the risk scoring engine in a single {kib} space within a cluster.
 
 * The risk scoring engine uses an internal user role to score all hosts and users, and doesn't respect privileges applied to custom users or roles. After you turn on the risk scoring engine for a {kib} space, all alerts in the space will contribute to host and user risk scores.
+
+[discrete]
+== Asset criticality
+
+To use the asset criticality feature, turn on the `securitySolution:enableAssetCriticality` <<enable-asset-criticality, advanced setting>>.
+
+[discrete]
+=== Privileges
+
+* To view an entity's asset criticality, you need the `read` privilege for the `.asset-criticality.asset-criticality-<space-id>` index.
+
+* To view, assign, or change an entity's asset criticality, you need the `read` and `write` privileges for the `.asset-criticality.asset-criticality-<space-id>` index.
```

Comment:
@elastic/security-entity-analytics, are there any cluster or Kibana privileges that we need to list here?

Reply:
cc: @jaredburgettelastic @SourinPaul @oatkiller see above please ⬆️

Reply:
No additional cluster or Kibana privileges are required, only the index-level privileges you have described above.